index.html

```diff
@@ -0,0 +1,237 @@
+				</div>
+				<!-- Card -->
+				<div class="card format">
+					<div class="card-content">
+						<h3 class="work-title">My Work Title</h3>
+						<p class="work-description">
+							I can help you to build beautiful projects, modern and responsive
+							websites, you can see my work on this page, if you are interested
+							do not hesitate to contact me
+						</p>
+						<ul class="categories">
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</div>
+					<button class="see-project-btn">See Project</button>
+				</div>
+				<!-- Card -->
+				<div class="card format">
+					<div class="card-content">
+						<h3 class="work-title">My Work Title</h3>
+						<p class="work-description">
+							I can help you to build beautiful projects, modern and responsive
+							websites, you can see my work on this page, if you are interested
+							do not hesitate to contact me
+						</p>
+						<ul class="categories">
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</div>
+					<button class="see-project-btn">See Project</button>
+				</div>
+				<!-- Card -->
+				<div class="card format">
+					<div class="card-content">
+						<h3 class="work-title">My Work Title</h3>
+						<p class="work-description">
+							I can help you to build beautiful projects, modern and responsive
+							websites, you can see my work on this page, if you are interested
+							do not hesitate to contact me
+						</p>
+						<ul class="categories">
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</div>
+					<button class="see-project-btn">See Project</button>
+				</div>
+				<!-- Card -->
+				<div class="card format">
+					<div class="card-content">
+						<h3 class="work-title">My Work Title</h3>
+						<p class="work-description">
+							I can help you to build beautiful projects, modern and responsive
+							websites, you can see my work on this page, if you are interested
+							do not hesitate to contact me
+						</p>
+						<ul class="categories">
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</div>
+					<button class="see-project-btn">See Project</button>
+				</div>
+				<!-- Card -->
+				<div class="card format">
+					<div class="card-content">
+						<h3 class="work-title">My Work Title</h3>
+						<p class="work-description">
+							I can help you to build beautiful projects, modern and responsive
+							websites, you can see my work on this page, if you are interested
+							do not hesitate to contact me
+						</p>
+						<ul class="categories">
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</div>
+					<button class="see-project-btn">See Project</button>
+				</div>
+			</div>
+		</section>
+		<section class="about-me" id="about-me">
+			<div class="resume">
+				<h1>About me</h1>
```

Comment:
- [ ] [OPTIONAL] Please, always use the `<h1>` tag only 1️⃣ once in the entire document 📄. Since [it represents the title of the document](https://html.com/tags/heading/) 👍.